### PR TITLE
Nerfs smoke machine + improves grammar

### PR DESCRIPTION
**Pull request by praisenarsie**
:cl: Frozenguy5
balance: Smoke machine has a maximum range of 9 now, as opposed to 15.
balance: Smoke machine has a maximum capacity of 300 reagents now, as opposed to 1000.
/:cl:

i had to redo pr plus this is to remind the maintainers this pr exists

### CODE REVIEW

File: code/modules/reagents/chemistry/machinery/smoke_machine.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/machinery/smoke_machine
-	name = "Smoke Machine"
-	desc = "Seriously man?"
+	name = "smoke Machine"
```
Comment (ShizCalev) [inline code/modules/reagents/chemistry/machinery/smoke_machine.dm]: Consistency please

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/machinery/smoke_machine.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+	desc = "A machine installed with a centrifuge inside of it. It generates smoke with whatever reagent you put into it."
```
Comment (ShizCalev) [inline code/modules/reagents/chemistry/machinery/smoke_machine.dm]: With a certifuge installed

Also if you wouldn't mind revising the last scentence a bit, it seems awkwardly worded.

**Review by ShizCalev — DISMISSED** [review]

**Comment by davethwave** [comment]
How about having the type of matter bin and so on and so forth you put in matter?

**Comment by Doppeltongue** [comment]
Matter bins for capacity, manipulators for efficiency, and capacitors for range, instead of all 3 being for efficiency right now?

**Comment by davethwave** [comment]
Yeah

**Comment by praisenarsie** [comment]
robustin can code it, i'm not touching his code.

**Comment by KieranCat** [comment]
Yes, god yes. No more gargle blaster/plasma/cyanide/deathmix/stun hell smoke in escape/escape shuttle that lasts forever.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Make the thing use components already. Give it the `fuck everything` range only if it has bluespace laser/manipulater. and the `fuck everything` volume if it's matter bin is bluespace

**Comment by praisenarsie** [comment]
i hate you cyberboss

**Pull request merged**